packages/shared/lib/seeders/connection.seeder.ts

```diff
@@ -19,40 +19,52 @@ export const createConnectionSeeds = async (env: DBEnvironment): Promise<number[
             environmentId: env.id,
             accountId: 0
         });
-        connectionIds.push(...result.map((res) => res.connection.id!));
+
+        for (const res of result) {
+            if (!res.connection.id) {
```

Comment:
Note this is just a type error not an actual error that can happen

Reply:
Yeah, I was just trying to get eslint to stop fussing at me. I figure if we're going to have warnings on we should structure the code such that they don't show up.

Reply:
I think most importantly we are going to fix the type at some point